cms/static/js/views/show_textbook.js

```diff
@@ -29,7 +30,7 @@ define(['js/views/baseview', 'underscore', 'gettext', 'common/js/components/view
                     if (e && e.preventDefault) { e.preventDefault(); }
                     var textbook = this.model;
                     new PromptView.Warning({
-                        title: _.template(gettext('Delete “<%= name %>”?'))(
+                        title: _.template(gettext('Delete “<%- name %>”?'))(
```

Comment:
I am actually not the most familiar with the syntax here, can you help me understand what the change from `=` to `-` does in this case?

Reply:
@fysheets This line also wasn't in my original PR, but the ESLint tests failed on it (the log is below), and I made this change in order to pass the tests.
The use of `<%-` (html-escaped strings) is safer than `<%=` in terms of injections. 
Here is a page of Underscore.js docs covering the use of `<%=  %>` and `<%-  %>`
https://underscorejs.org/#template
Log:
```
Linting cms/static/js/views/show_textbook.js:
cms/static/js/views/show_textbook.js: 22:29: javascript-jquery-html:                     this.$el.html(this.template(attrs));

cms/static/js/views/show_textbook.js: 33:60: underscore-not-escaped:                         title: _.template(gettext('Delete “<%= name %>”?'))(

2 violations total
```